apps/arena/lib/arena/matchmaking/game_launcher.ex

```diff
@@ -51,16 +29,24 @@ defmodule Arena.Matchmaking.GameLauncher do
   def handle_call({:join, client_id, character_name, player_name}, {from_pid, _}, %{clients: clients} = state) do
     batch_start_at = maybe_make_batch_start_at(state.clients, state.batch_start_at)
 
+    client = %{
+      client_id: client_id,
+      character_name: character_name,
+      name: player_name,
+      from_pid: from_pid,
+      type: :human
```

Comment:
Is there a reason for this `type` parameter? just wondering

Reply:
It's used by the game_updater genserver to save human and bot ids in different lists.
See game_updater.ex:80-84